cas-server-integration-memcached/src/main/resources/META-INF/spring/memcached-registry-config.xml

```diff
@@ -37,7 +37,7 @@
        http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
 
         <bean id="memcachedClient" class="net.spy.memcached.spring.MemcachedClientFactoryBean"
-              p:servers="${memcached.servers:cas-1.example.org:11211}"
+              p:servers="${memcached.servers:localhost:11211}"
```

Comment:
I also changed the default Memcached server to be `localhost` which works out of the box for a local Memcached on a development workstation.